txbridge/src/main/java/org/jboss/jbossts/txbridge/outbound/BridgeXAResource.java

```diff
@@ -87,10 +88,9 @@ public BridgeXAResource(Uid externalTxId, BridgeWrapper bridgeWrapper)
      */
     private void writeObject(ObjectOutputStream out) throws IOException
     {
-        txbridgeLogger.logger.trace("BridgeXAResource.writeObject() for tx id="+externalTxId);
```

Comment:
@mmusgrov ok, just I'm unsure - should I change it back to trace logging (which I think is good idea) + add the check if trace is enabled?